crates/db_views_actor/src/structs.rs

```diff
@@ -149,3 +149,12 @@ pub struct PersonView {
   pub counts: PersonAggregates,
   pub is_admin: bool,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[cfg_attr(feature = "full", derive(Queryable))]
+#[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
+/// A site person ban.
+pub struct SitePersonBanView {
+  pub site: Site,
+  pub person: Person,
+}
```

Comment:
The view might not be necessary, since we're not returning this view specifically, but only fields joined from `site_person_ban`

Same for `CommunityPersonBanView`, but that's up to you whether to remove it or not.